BookingsAPI/Bookings.DAL/Commands/HearingService.cs

```diff
@@ -12,21 +14,29 @@ namespace Bookings.DAL.Commands
     public interface IHearingService
     {
         /// <summary>
-        /// Add a participant to a hearing service. This will re-use existing personnel entries before attempting to
-        /// create a new one.
+        /// Add a participant to a hearing service and returns a list of all participants.
+        /// This will re-use existing personnel entries before attempting to create a new one.
         /// </summary>
         /// <param name="hearing">Hearing to amend</param>
         /// <param name="participants">List of participants to add</param>
         /// <returns></returns>
-        Task AddParticipantToService(VideoHearing hearing, List<NewParticipant> participants);
+        Task<List<Participant>> AddParticipantToService(VideoHearing hearing, List<NewParticipant> participants);
 
         /// <summary>
         /// Update the case name of a hearing directly
         /// </summary>
         /// <param name="hearingId">Id of hearing</param>
-        /// <param name="caseName">new case name</param>
+        /// <param name="caseName">New case name</param>
         /// <returns></returns>
         Task UpdateHearingCaseName(Guid hearingId, string caseName);
+        
+        /// <summary>
+        /// Create links between participants if any exist
+        /// </summary>
+        /// <param name="participants">List of participants in the hearing</param>
+        /// <param name="requests">List of requests containing information of any participant links</param>
+        /// <returns></returns>
+        Task<List<LinkedParticipant>> CreateParticipantLinks(List<Participant> participants, List<LinkedParticipantDto> linkedParticipantDtos);
```

Comment:
Why do we return a list of linked participants here?

Reply:
Because once the links have been created for the participants, we still need to write the corresponding rows to the DB. This is done by these returned `List<LinkedParticipant>`.